doc/build.xml

```diff
@@ -92,8 +92,7 @@
              linkoffline="http://download.oracle.com/javaee/6/api/ validation-package-list">
       <!-- suppress errors that don't break the output, we should refine this to just suppress things we haven't fixed yet -->
       <arg value="-Xdoclint:none" />
-      <!-- re-enable frames for now, until we get to >=13 and have search -->
-      <arg value="--frames" />
+
```

Comment:
We should consider leaving this in for the next release so that we can get similar looking output? Or maybe not, and jump all the way to 17 for this next build?

Reply:
It fails to build with flag on 17.

Reply:
flag restored